students/srmorehouse/Lesson03/slicing_lab.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+"""
+Steve Morehouse
+Lesson 03
+
+"""
+
+"""
+Write some functions that take a sequence as an argument, and return a copy of that sequence:
+with the first and last items exchanged.
+"""
+def exchange_first_last ( seq ):
+    return seq[-1:] + seq[1:len(seq)-1] + seq[:1]
+
+"""
+Write some functions that take a sequence as an argument, and return a copy of that sequence:
+with every other item removed.
+"""
+def remove_every_other ( seq ):
+    return seq[::2]
+
+"""
+Write some functions that take a sequence as an argument, and return a copy of that sequence:
+with the first 4 and the last 4 items removed, and then every other item in the remaining sequence.
+"""
+def remove_first_last_four ( seq ):
+    return seq [4:-4:2]
+
+"""
+Write some functions that take a sequence as an argument, and return a copy of that sequence:
+with the elements reversed (just with slicing).
+"""
+def reverse_sequence ( seq ):
+    return seq [::-1]
```

Comment:
note that there is no space between your variable and slice syntax -> `return seq[::-1]`